### DISCUSSION TITLE
Clang "Error: Process completed with exit code 1."

### DISCUSSION THREAD

**Discussion by IanBUK** [community/community#26540] [Actions]
<p>Builds done when I submit a pull request to the Adafruit_GFX library fail with “Error: Process completed with exit code 1.” but no indication of what caused the failure. The code builds and runs locally.  I’ve tried:</p>
<ol>
<li>Convert to unix line endings.</li>
<li>Move to a different branch.</li>
</ol>
<p>It still fails, and I’ve no indication why.  Any suggestions on what I could look at?</p>
<p>Thanks.</p>

**Comment by ernest-phillips** [comment]
<p>Hi <a class="mention" href="/ianbuk">@IanBUK</a> :wave:</p>
<p>Welcome to the GitHub community forum!<br>
I moved your post to our Actions board as it looks to be a better fit here.</p>
<p>Could you provide us with a link to the repository in question here?</p>

**Comment by IanBUK** [comment, marked as answer]
<p><a class="mention" href="/ernest-phillips">@ernest-phillips</a> thanks for getting in touch. I found the solution. I was getting clang failures and as I’m new to GitHub thought that meant my code was failing to compile. Now I know it was styling failures. I’ve fixed those - and the Doxygen errors.</p>
<p>Thanks.</p>
